### PR TITLE
Shopify new updates

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: stream-sources/shopify.mdx [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+</div>
+
+6. After installation, you should be able to see the app in the **Installed apps** section, as shown:
+
+<img src="../assets/event-stream-sources/shopify-5.png" alt="Installed RudderStack app" />
+
+6. Select the RudderStack app.
+7. Enter your **RudderStack Data Plane URL** and the **Source Write Key** that you copied [**above**](#setting-up-the-shopify-source-in-rudderstack), as shown:
+
+<img src="../assets/event-stream-sources/shopify-6.png" alt="Data plane URL and source write key" />
```
Comment (saikumarrs) [inline stream-sources/shopify.mdx:70]: The image used here does not seem latest
CC @yashasvibajpai

File: faqs.mdx [hunk 2]
```diff
@@ -222,7 +222,7 @@ You should use the `track` method in the SDK instead. For JavaScript SDK's `trac
 
 #### 2. Is Shopify compatible as a data source for Rudderstack?
 
-Currently we do not have an integration for Shopify as an event stream source. However, we do have users that integrate the JavaScript SDK into their Shopify sites. In some cases, they even do it through Google Tag Manager.
+Yes, Shopify is compatible as a data source. For more information, refer [Shopify](https://www.rudderstack.com/docs/stream-sources/shopify/).
```
Comment (saikumarrs) [inline faqs.mdx:225]: Let's also keep the other methods of integrations intact. However, strongly recommend the Shopify source option.

**Review by saikumarrs — CHANGES_REQUESTED** [review]
LGTM

File: stream-sources/shopify.mdx (hunk 1 above)
Comment (ishachopra0401) [reply stream-sources/shopify.mdx:70]: Updated the image.

File: faqs.mdx (hunk 2 above)
Comment (ishachopra0401) [reply faqs.mdx]: Amended this section.

File: faqs.mdx [hunk 3]
```diff
@@ -222,7 +222,7 @@ You should use the `track` method in the SDK instead. For JavaScript SDK's `trac
 
 #### 2. Is Shopify compatible as a data source for Rudderstack?
 
-Currently we do not have an integration for Shopify as an event stream source. However, we do have users that integrate the JavaScript SDK into their Shopify sites. In some cases, they even do it through Google Tag Manager.
+Yes, Shopify is compatible as a data source. For more information, refer [Shopify](https://www.rudderstack.com/docs/stream-sources/shopify/). However, we also have users that integrate the JavaScript SDK into their Shopify sites. In some cases, they even do it through Google Tag Manager.
```
Comment (saikumarrs) [inline faqs.mdx:225]: Recommended Shopify source.
```suggestion
Yes, Shopify is compatible as an event stream data source. For more information, refer to [Shopify](https://www.rudderstack.com/docs/stream-sources/shopify/). We also have users that integrate the JavaScript SDK into their Shopify sites. In some cases, they even do it through Google Tag Manager.
However, we strongly recommend using the Shopify source integration for better tracking.
```

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**